pkg/services/object/server.go

```diff
@@ -495,15 +497,16 @@ func (s *server) Delete(ctx context.Context, req *protoobject.DeleteRequest) (*p
 	)
 	defer func() { s.pushOpExecResult(stat.MethodObjectDelete, err, t) }()
 
-	if err = icrypto.VerifyRequestSignatures(req); err != nil {
+	author, authorPub, err := icrypto.AuthenticateRequest(req)
+	if err != nil {
 		return s.makeStatusDeleteResponse(err), nil
 	}
 
 	if s.fsChain.LocalNodeUnderMaintenance() {
 		return s.makeStatusDeleteResponse(apistatus.ErrNodeUnderMaintenance), nil
 	}
 
-	reqInfo, err := s.reqInfoProc.DeleteRequestToInfo(req)
+	reqInfo, err := s.reqInfoProc.DeleteRequestToInfo(req, author, authorPub)
```

Comment:
This passing of data around doesn't seem elegant.

Reply:
redone